### ISSUE TITLE
CoreOS v1632.2.1 - [STABLE] cloud-Init removed with no warning?

### ISSUE THREAD

**Issue by qrpike** [coreos/bugs#2353]
# Issue Report #
I recently updated to v1632.2.1 [STABLE] branch, and cloud-init just stopped working.

There was no warning of this in the release notes, and is a pretty important thing to remove from a stable release without warning.

### Container Linux Version ###
1632.2.1

### Environment ###
PXE Boot to read-only coreos 

What hardware/cloud provider/hypervisor is being used to run Container Linux?
Bare Metal Dell 620's

### Expected Behavior ###
Loads cloud-init from url

### Actual Behavior ###
Does not attempt to load cloud-init 

### Reproduction Steps ###
Updade to latest stable version, from any version before that.

**Comment by lucab** [comment]
Can you please check and attach more details from your system journal? From a quick look, it looks like this may be a duplicate of  https://github.com/coreos/bugs/issues/2342.

**Comment by qrpike** [comment]
I can't heh.

The SSH keys are on the cloud-init file, which it doesn't load.

I look at the http logs on the cloud-init server, and no request was made. I downgrade to the previous version and it works fine. I also validated the cloud-init file to make sure there were no errors in it.

**Comment by ajeddeloh** [comment]
You could try downgrading to the previous version, then writing
```
[Unit]
ConditionVirtualization=!container
ConditionPathExists=!/usr/.noupdate
```
 to `etc/system/systemd-fsck@dev-disk-by\x2dlabel-OEM.service.d/conditional.conf` (essentially manually applying https://github.com/coreos/init/commit/e8febaa0ffb0cdfbdc9accaf8296f2cd476aad95)

That would tell you if it is in fact the oem fsck bug, and work around it if it is.

**Comment by bgilbert** [comment]
@ajeddeloh That won't work unless @qrpike is using a persistent root filesystem.

@qrpike You should be able to add `coreos.autologin` to the PXE command line and get access to the journal from the system console, or add `sshkey="ssh-rsa ..."` to the command line and then SSH to the machine.

**Comment by bgilbert** [comment]
I'm going to optimistically assume this is a duplicate of #2342.  If so, it should be fixed in stable 1632.3.0, which is due shortly.  If you still see this after updating to 1632.3.0, please reopen the bug.
